### PR TITLE
paint_board-8: Created component footer and added outside fonts

### PR DESCRIPTION
**Pull request by KoTuK2306**

### CODE REVIEW

File: src/components/footer/footer.style.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { createUseStyles } from 'react-jss';
+import color from 'color';
+
+import { Colors } from '../../style-constants';
+import '../../fonts/Montserrat/Montserrat.css';
```
Comment (pavelsokolov21) [inline src/components/footer/footer.style.tsx]: just `fonts/...`

File: src/components/footer/footer.style.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { createUseStyles } from 'react-jss';
+import color from 'color';
+
+import { Colors } from '../../style-constants';
```
Comment (pavelsokolov21) [inline src/components/footer/footer.style.tsx]: just `style-constants`

File: src/components/footer/footer.style.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+import '../../fonts/Montserrat/Montserrat.css';
+
+export const Styles = createUseStyles({
+  footer: {
+    background: color(Colors.gray).darken(0.75).hex(),
+  },
+  footerText: {
+    padding: 10,
+    color: color(Colors.gray).darken(0.45).hex(),
+    fontFamily: 'Montserrat-Regular',
```
Comment (pavelsokolov21) [inline src/components/footer/footer.style.tsx]: move right after padding

File: src/components/footer/footer.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { FC } from 'react';
+
+import { Styles } from './footer.style';
+
+export const Footer: FC = () => {
+  const classes = Styles();
+
+  return (
+    <footer className={classes.footer}>
+      <p className={classes.footerText}>© Paint board 2021</p>
```
Comment (pavelsokolov21) [inline src/components/footer/footer.tsx]: set dynamic year

File: src/fonts/Montserrat/Montserrat.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
+@font-face {
+  font-family: 'Montserrat-Bold';
```
Comment (pavelsokolov21) [inline src/fonts/Montserrat/Montserrat.css:2]: create enum with all fonts and set default `sans-serif`

File: src/components/footer/footer.style.tsx (hunk 1 above)
Comment (pavelsokolov21) [reply src/components/footer/footer.style.tsx]: move this import to index.css

**Review by pavelsokolov21 — CHANGES_REQUESTED** [review]

**Review by pavelsokolov21 — APPROVED** [review]

File: .eslintrc [hunk 6]
```diff
@@ -14,6 +14,8 @@
     "react/react-in-jsx-scope": 0,
     "react/jsx-uses-react": 0,
     "react-hooks/rules-of-hooks": "error",
+    "no-shadow": "off",
```
Comment (IliaSolovev) [inline .eslintrc:17]: delete this rules

File: .eslintrc [hunk 7]
```diff
@@ -14,6 +14,8 @@
     "react/react-in-jsx-scope": 0,
     "react/jsx-uses-react": 0,
     "react-hooks/rules-of-hooks": "error",
+    "no-shadow": "off",
+    "@typescript-eslint/no-shadow": "error",
```
Comment (IliaSolovev) [inline .eslintrc:18]: and this

File: src/components/footer/footer.tsx [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import { FC } from 'react';
+
+import { Styles } from './footer.style';
+
+export const Footer: FC = () => {
+  const classes = Styles();
+
+  return (
+    <footer className={classes.footer}>
+      <p className={classes.footerText}>© Paint board {new Date().getFullYear()}</p>
```
Comment (IliaSolovev) [inline src/components/footer/footer.tsx]:
```suggestion
      <p className={classes.text}>© Paint board {new Date().getFullYear()}</p>
```

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

**Review by IliaSolovev — APPROVED** [review]

**Pull request merged**